examples/publication/valid/bibcast16.json

```diff
@@ -35,6 +35,6 @@
     "web developers",
     "librarians"
   ],
-  "isBasedOn": "http://hbz.github.io/slides/swib-15/",
+  "isBasedOn": [{"id": "http://hbz.github.io/slides/swib-15/"}],
```

Comment:
I think we should unify them all not to array of object, but to array of string (i.e. `"isBasedOn": ["http://hbz.github.io/slides/swib-15/"]` here).